docs/labels.md

```diff
@@ -0,0 +1,106 @@
+# GitHub Label List
+
+We use GitHub labels to perform issue triage, track and report on development
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/conformance                   | Issues or PRs related to conformance testing | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/e2e-test                      | Issues or PRs related to end-to-end testing. | Any |
```

Comment:
is this we only thing we have for "testing", or should we distinguish between tests themselves and CI infra?

Reply:
we need some additional area labels for tests. I will update accordingly.